src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -535,6 +535,7 @@ bool generate_ansi_c_start_function(
   symbolt new_symbol;
```

Comment:
💡 Maybe a proper constructor could help.

Reply:
We'd just need to get #6591 approved (@kroening ?), which not only adds constructors, but also changes this line of code to use one of those constructors!